core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -8247,6 +8343,33 @@ private static String sums(int n, boolean c) {
     return b.toString();
   }
 
+  /** A program that omits {@link TrimFieldsProgram}. */
+  private static Program getProgram() {
```

Comment:
Maybe could use org.apache.calcite.tools.Programs#standard(org.apache.calcite.rel.metadata.RelMetadataProvider) method.

Reply:
The standard program is initialized as follows:
```
    return sequence(subQuery(metadataProvider),
        new DecorrelateProgram(),
        new TrimFieldsProgram(),
        program1,

        // Second planner pass to do physical "tweaks". This the first time
        // that EnumerableCalcRel is introduced.
        calc(metadataProvider));
```
Which includes the `TrimFieldsProgram()`, so I had to manually initialize one to avoid adding that program.
What do you think? Do you suggest a better way to create the program without `TrimFieldsProgram()`?

Reply:
Could we make the standard(RelMetadataProvider) method has parameters with RelMetadataProvider and `boolean` flag?

The `boolean` flag  indicates whether TrimFieldsProgram should be added,The standard() method default value is true.I finded Programs#standard(RelMetadataProvider) method only used in Programs.

Reply:
That sounds great! Will go for that!